test/Test/Database/LSMTree/Generators.hs

```diff
@@ -62,6 +65,11 @@ tests = testGroup "Test.Database.LSMTree.Generators" [
         prop_arbitraryAndShrinkPreserveInvariant
           labelRunData
           (const True)  -- no invariant
+     ++ [ testProperty "withRun doesn't leak resources" $ \rd ->
+            QC.ioProperty $
+              withSimHasBlockIO propNoOpenHandles MockFS.empty $ \hfs hbio _ ->
+                prop_withRunDoesntLeak hfs hbio rd
+        ]
```

Comment:
I'd use the name of the property as the test name, i.e., `prop_withRunDoesntLeak`, because it makes it easier to run them from the command line. Similarly for other properties below

Reply:
I followed `prop_arbitraryAndShrinkPreserveInvariant`, so the test output looks consistent. We can change it there, too, of course.

Reply:
I probably should have done it in #604, won't do it here.